web/src/main/java/org/cbioportal/genome_nexus/GenomeNexusAnnotation.java

```diff
@@ -47,18 +47,16 @@
 import springfox.documentation.spring.web.plugins.Docket;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
-import java.io.*;
 import java.util.Arrays;
 import java.util.HashSet;
 
 /**
  * @author Benjamin Gross
  */
-@SpringBootApplication // shorthand for @Configuration, @EnableAutoConfiguration, @ComponentScan
+@SpringBootApplication(scanBasePackages = {"org.cbioportal.genome_nexus"}) // shorthand for @Configuration, @EnableAutoConfiguration, @ComponentScan
```

Comment:
`scanBasePackages = {"org.cbioportal.genome_nexus"}` required due to multiple modules